README.md

```diff
@@ -102,3 +92,21 @@ You can also include Ace editor directly from a location that follow the same di
 ace_editor:
     base_path: "http://rawgithub.com/ajaxorg/ace-builds/master"
 ```
+
+
+## Ace editor assets
+
+Unless you do some configuration, this bundle expects Ace editor files to be in `public/vendor/ace`.
```

Comment:
One thing I'm thinking about is that it would be cool to also prepare this bundle to work together with assets mapper, then ace editor dependency could be handled by it 😁